build.gradle

```diff
@@ -196,17 +196,21 @@ task fastbuild(dependsOn: [
     ':dbfit-java:hsqldb:integrationTest'
     ])
 
-task integrationBuild(dependsOn: [
-    ':dbfit-java:db2:integrationTest',
-    ':dbfit-java:derby:integrationTest',
-    ':dbfit-java:hsqldb:integrationTest',
-    ':dbfit-java:mysql:integrationTest',
-    //':dbfit-java:netezza:integrationTest',
-    ':dbfit-java:oracle:integrationTest',
-    ':dbfit-java:postgres:integrationTest'
-    //':dbfit-java:sqlserver:integrationTest',
-    //':dbfit-java:teradata:integrationTest',
-    ])
+def integrationBuildTasks = []
+def excludedIntegrationBuildDbTypes = []
+
+if (project.hasProperty('excludeIntegrationTests')) {
+    excludedIntegrationBuildDbTypes=project.excludeIntegrationTests.split(",")
+}
+
+subprojects.each { sp ->
+    if (!(sp.name in ["dbfit-java", "core"])) {
+        sp.name in excludedIntegrationBuildDbTypes ?
```

Comment:
This may be defined as set operation, i.e. something like
``` groovy
(subprojects.collect { it.name } - excludedIntegrationBuildTypes).collect {
    ":dbfit-java:${it}:integrationTest"
}
```